internal/pkg/tmpl/template.go

```diff
@@ -160,6 +160,24 @@ func builtinFuncs(settings *print.Settings) map[string]interface{} {
 		"sanitizeTbl": func(s string) string {
 			return markdown.SanitizeItemForTable(s, settings)
 		},
```

Comment:
@khos2ow How do you feel about the below changes? This is how I was able to keep the type in the generated print out. If there are any cases I have forgotten, let me know!

Reply:
I'm gonna take a closer look, but by the look of it it seems great. Although I'm also working on another PR to streamline the use of "default values" and not actually depend on `interface{}` as a literal. I'm gonna have to combine these change of yours and mine together to see if we would need both or either. I'm gonna circle back on this soon.

Reply:
As my above comment, I think this is good for now and we can take required actions if needed in #196 after rebase.